includes/features/drupal.org/user/adminstrator_role.feature

```diff
@@ -3,7 +3,7 @@ Feature: module
   As administrator
   I want to verify permissions and access
 
-  @safe @api
+  @safe @api @core
```

Comment:
I think I like what you have earlier and keeping these tasks tagged with lifecycle not what.